packages/yourloops/test/unit/lib/patient/patient.api.test.ts

```diff
@@ -66,6 +67,34 @@ describe('PatientApi', () => {
     })
   })
 
+  describe('getPatient', () => {
+    it('should get a patient', async () => {
+      const userId = '1234'
+      const data: Patient = { userid: userId } as Patient
+      jest.spyOn(HttpService, 'get').mockResolvedValueOnce({ data } as AxiosResponse)
+
+      const patients = await PatientApi.getPatient(userId)
+
+      expect(patients).toEqual([data])
+      expect(HttpService.get).toHaveBeenCalledWith({ url: `/bff/v1/patients/${userId}` })
+    })
+
+    it('should return an empty array if not found', async () => {
+      jest.spyOn(HttpService, 'get').mockRejectedValueOnce(Error(ErrorMessageStatus.NotFound))
+      const userId = '1234'
+      const response = await PatientApi.getPatient(userId)
+      expect(response).toBeInstanceOf(Array)
```

Comment:
This is not testing that the arry is empty
```suggestion
      expect(response).toEqual([])
```